<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodePaster::Internal::SettingsPage</class>
 <widget class="QWidget" name="CodePaster::Internal::SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>229</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>223</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="displayBox">
     <property name="text">
      <string>Display Output pane after sending a post</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="clipboardBox">
     <property name="text">
      <string>Copy-paste URL to clipboard</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="userEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="userNameLabel">
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="defaultProtocol"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="protocolLabel">
     <property name="text">
      <string>Default protocol:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
